docs/plugins/alexa-plugin.md

```diff
@@ -242,108 +89,67 @@ After you enable testing, you can also use the Alexa Simulator in the left colum
 
```

Comment:
Indicate on the above lines somewhere (line 235-241). You can not type the keyword "Alexa" in the devloper console Test tab. This will confuse the alexa simulator. Instead just type out "Ask night scout how I am doing"

Reply:
Not a bad idea...

Reply:
It took me two days to figure this out. I believe if you use a mic and ask  "Alexa, ask night scout how I am doing" It also gets confused. I was just copying and pasting from the suggested commands, which does not work.